.github/workflows/setup-and-test.yml

```diff
@@ -24,8 +24,8 @@ jobs:
           yarn set version 3.x
           yarn install
 
-      - name: Run tests
-        run: CI=true yarn test
+      - name: Run linter
+        run: yarn eslint --max-warnings=0 .
```

Comment:
Should this be reverted back to running tests?

Reply:
🤦‍♂️ thank you

Reply:
And _that's_ why I re-request reviews 😄